mat64/dense.go

```diff
@@ -475,6 +514,9 @@ func (m *Dense) TCopy(a Matrix) {
 	*m = w
 }
 
+// Stack stacks a and b, where b is placed in the greater indexed rows.
```

Comment:
Stack appends the rows of b onto the rows of a, placing the result into the receiver.